src/@next/components/organisms/Overlay/styles.ts

```diff
@@ -0,0 +1,83 @@
+import { styled } from "@styles";
+import { css, keyframes } from "styled-components";
+import { Position, TransitionState } from "./types";
+
+interface IStyleProps {
+  open: boolean;
+  position: Position;
+  state: TransitionState;
+}
+
+const slideAnimation = (open: boolean, side: "left" | "right") => {
+  const initialValue = open ? "-100rem" : 0;
```

Comment:
> -100rem

We want the element to be completely off the screen, or am I looking at this wrong? 100% is more safe and easier to control its position, some huge resolutions may still show this element at -100rem.